blas/testblas/common.go

```diff
@@ -276,6 +276,28 @@ func zsame(x, y []complex128) bool {
 	return true
 }
 
+func zequal(x, y []complex128, tol float64) bool {
```

Comment:
It seems that this is more `zsameApprox` rather than `zequal`.

Reply:
I changed the signature and the semantics of this function. I think it is better now.